public/app/features/alerting/unified/utils/receiver-form.ts

```diff
@@ -292,3 +292,15 @@ export function omitEmptyValues<T>(obj: T): T {
 export function omitEmptyUnlessExisting(settings = {}, existing = {}): Record<string, unknown> {
   return omitBy(settings, (value, key) => isUnacceptableValue(value) && !(key in existing));
 }
+
+export function omitTemporaryIdentifiers<T>(obj: T): T {
```

Comment:
Could we make this functional/pure, so it doesn't mutate the object that's been passed in? Doesn't seem to matter at the moment, but it feels like it could be confusing in the future